config/test_assets/good_config.yml

```diff
@@ -33,6 +33,7 @@ cf:
       authorized_grant_types: grant_type1,grant_type2
       resource_ids: resource2,resource3
       name: client_name
+      allowpublic: true
```

Comment:
`allow_public` for consistency instead of `allowpublic`?

Reply:
I kept it consistent with the field name in UAA https://docs.cloudfoundry.org/api/uaa/version/76.6.0/index.html#create-6

I actually agree with you and wish the field name was instead "allow_public"!#189 

Shall I change it to allow_public so the field names are consistent in ODB?